utils/efushell/SocketDriver.py

```diff
@@ -6,30 +6,32 @@
 
 
 class SimpleSocket:
-  def __init__(self, hostname = "localhost", port = 8888, timeout = 2):
-    self.access_semaphor = threading.Semaphore(1)
-    try:
-        self.sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
-    except socket.error, msg:
-        sys.stderr.write("socket() [Socket connection error] Cannot connect to %s, error: %s\n" % (hostname, msg[0]))
-        sys.exit(1)
+    def __init__(self, hostname="localhost", port=8888, timeout=2):
```

Comment:
Looks like re-formatting only?
Do we have a guideline for python formatting and a tool for doing it?

Reply:
Ideally we'd use [black](https://github.com/ambv/black) for formatting but it requires Python 3.6. It might be that we could move to Python 3.6 on CentOS now. I'll be spending some time on a Python project in the coming weeks, so I'll talk with Afonso about that.
PyCharm did the changes in this PR. I hit the formatting hotkey because I noticed a mixture of 2 and 4 spaces used for indentation.